templates/zerver/help/private-messages.md

```diff
@@ -9,38 +9,44 @@ frequently conversing with the same person or the same group of people, we
 highly recommend [creating a private stream](/help/create-a-stream) for your
 conversations instead.
 
-Note that group PMs are not the same as [user groups](/help/user-groups) or private
-streams. PMs and group PMs do not have
-[topics](/help/streams-and-topics), and do not appear in your list of
-streams.
+Note that group PMs are not the same as [user groups](/help/user-groups) or
+private streams. PMs and group PMs do not have [topics](/help/streams-and-topics)
+and do not appear in your list of streams.
 
 ### Send a PM or group PM
 
 {start_tabs}
 
-1. [Open the compose box](/help/open-the-compose-box) by typing `x` or clicking
-   **New private message** at the bottom of the screen.
+{tab|desktop-web}
 
-2. Start typing a user's name, and select the user from the list of
+1. Use the `x` keyboard shortcut to
+   [Open the compose box](/help/open-the-compose-box) or  
+   click **New private message** at the bottom of the screen.
+
```

Comment:
Let's change this to:

> To start composing a private message, click the **New private message** button at the bottom of the Zulip window, or use the `x` keyboard shortcut.

I want to put the button UI before the power-user keyboard UI, and "bottom of the screen" doesn't seem right for web/desktop instructions, where the Zulip window might take up just part of the user's screen.

Reply:
Sounds good! But I would suggest dropping "To start composing a private message, ". The instruction would be more concise, it'd be consistent with other articles, and also the last step begins with "Compose your message,".

Reply:
And we can move the "compose box" link to the "Related articles" section.